TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  OrganizationInfoViewController.swift
+//  TutorialApplication
+//
+//  Created by Aleksandr Yegorov on 2/10/17.
+//  Copyright © 2017 Aleksandr Yegorov. All rights reserved.
+//
+
+import UIKit
+
+class OrganizationInfoViewController: UIViewController {
+
+    var organization: FFOrganization!
+
+    @IBAction func salarySumOffAllEmployees()
+    {
+        var salarySum = 0
+        
+        for obj in organization.employees!
```

Comment:
If you are referencing object's property, prefix it with self., so you should use self.organization, not just organization